run.sh

```diff
@@ -14,4 +14,8 @@ fi
 
 printf "running..."
 
-parallel ::: "cargo run -r" "ffmpeg -re -loop 1 -i ${1} -filter:v fps=30 -s 640x480 -f v4l2 -vcodec rawvideo -pix_fmt yuyv422 /dev/video42"
+if [ $(file -ib $1 | grep -c image) -eq 1 ]; then
```

Comment:
```suggestion
if [ ! -z "$(file -ib $1 | grep image)" ]; then
```

I think the more commonly used `! -z`, makes it a bit more obvious what we're testing.